test/docTest.js.html

```diff
@@ -135,6 +135,12 @@
             assert.equal($('#find-element').find('#sub-find-2').size, 1);
           });
 
+          it("should find the closest elements", function() {
+            assert.equal($('.sub-find').closest('.find').size, 2);
+            assert.equal($('#sub-find-2').closest('#find-element').size, 1);
+            assert.equal($('.find').closest('#find-element').size, 1);
+          });
+
```

Comment:
It will be nice to create a new test to verify if the `closest` function works when there is parent from the parent that matches the selector, example:

```html
<div>
   <div>
      <p id='selector'></p>
   </div>
</div>
```

```js
define(['doc'], function(doc) {
   doc('#selector').closest('div');  //It should return only one div.
});
```

@erikatakahara @williammizuta